src/components/pages/dashboard/panels/alarms/alarmsPanel.js

```diff
@@ -44,7 +44,7 @@ export class AlarmsPanel extends Component {
     return (
       <Panel className="alarms-panel-container">
         <PanelHeader>
-          <PanelHeaderLabel>{t('dashboard.panels.alarms.header')}</PanelHeaderLabel>
```

Comment:
Can you rename files as well otherwise it will be confusing. And class name to AlertsPanel